src/pager.c

```diff
@@ -706,14 +706,14 @@ struct permonst * pm;
     }
 #define MONPUTSTR(str) putstr(datawin, ATR_NONE, str)
 
+    Sprintf(buf, "Monster lookup for \"%s\":", pm->mname);
+    putstr(datawin, ATR_BOLD, buf);
+    MONPUTSTR("");
+
     /* Misc */
     Sprintf(buf, "%s", mname);
     putstr(datawin, ATR_INVERSE, mname);
 
-    Sprintf(buf, "Difficulty %d, speed %d, base AC %d, magic saving throw %d, weight %d.",
```

Comment:
why is this being removed?

Reply:
No clue, probably a misplaced <kbd>d</kbd><kbd>}</kbd>! Thanks!